demo/index.html

```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```

Comment:
I think I found the problem. It seems like the mapping(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/bc.js) is done for the full version(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/build.json) but not for lite version(https://github.com/webcomponents/webcomponentsjs/blob/master/src/WebComponents/build-lite.json)

Should I just use the full version of webcomponents on this file?

Is this something that needs to be fixed in the webcomponents repo? I can create a pull request for that if necessary.